core/base/inc/TError.h

```diff
@@ -26,8 +26,11 @@
 //////////////////////////////////////////////////////////////////////////
 
 
-#include "RtypesCore.h"
-#include <stdarg.h>
+#include "Rtypes.h"
```

Comment:
That is incorrect. RtypesCore.h allows us to include this header in -fno-rtti context.

In general, I am not quite sure if we should be doing such a header cleanup...

Reply:
I put here `Rtypes.h` to also access RConfigure.h immediately, but ok - one can do it explicitly. I will change it.